schemas/public/onpremises-kfd-v1alpha2.json

```diff
@@ -357,9 +365,11 @@
       "additionalProperties": false,
       "properties": {
         "name": {
+          "description": "A name to identify the host (not the hostname or FQDN)",
```

Comment:
This name is prefixed to the DNS to calculate the full fqdn of the VM, maybe we need to write something about it?

Reply:
```suggestion
          "description": "A name to identify the host. This value will be concatenated to `.spec.kubernetes.dnsZone` to calculate the FQDN for the host as `<name>.<dnsZone>`."
```